clang-tools-extra/docs/clang-tidy/checks/modernize/use-scoped-lock.rst

```diff
@@ -0,0 +1,100 @@
+.. title:: clang-tidy - modernize-use-scoped-lock
+
+modernize-use-scoped-lock
+=========================
+
+Finds uses of ``std::lock_guard`` and suggests replacing them with C++17's
+alternative ``std::scoped_lock``. The check will automatically transform only
```

Comment:
avoid "The check", documentation is already in context of check.
put empty line after this first sentence.